src/aleph_client/commands/account.py

```diff
@@ -115,3 +117,37 @@ def export_private_key(
 def path():
     if sdk_settings.PRIVATE_KEY_FILE:
         typer.echo(sdk_settings.PRIVATE_KEY_FILE)
+
+
+@app.command()
+def sign(
```

Comment:
What about calling this `sign_bytes` or something similar, to express that it is different than signing a message ?